runner/runner.go

```diff
@@ -365,6 +367,7 @@ func runAllInit() {
 	redshift.Init()
 	snowflake.Init()
 	deltalake.Init()
+	deltalake_native.Init()
```

Comment:
It'd be a good idea to start moving away from this pattern and stop having to call `Init` functions of packages which is known to create issues due to the use of global variables and due to the fact that we saw cases where the order of calling `Init` packages created unexpected outcomes.

In this case it's simple enough to fix given that in the `Init` you just create a logger. Perhaps for now we can move the logger creation in the constructor. That would allow us to remove both the global variable `pkgLogger` and the `Init` function.

Reply:
https://github.com/rudderlabs/rudder-server/pull/3202